internal/karma/karma.conf.js

```diff
@@ -110,6 +110,29 @@ resolveRunfiles(manifest, [
   files.push({ pattern: f, included: false });
 });
 
+var initializeTestbedContent = `
+// Try to import and initialize the Angular TestBed, if available.
+(function(){
+  try {
+    const getTestBed = require('@angular/core/testing').getTestBed;
+    const BrowserDynamicTestingModule = 
+      require('@angular/platform-browser-dynamic/testing').BrowserDynamicTestingModule;
+    const platformBrowserDynamicTesting = 
+      require('@angular/platform-browser-dynamic/testing').platformBrowserDynamicTesting;
+    
+    getTestBed().initTestEnvironment(
+      BrowserDynamicTestingModule,
+      platformBrowserDynamicTesting()
+    );
+  } catch (e) { }
```

Comment:
even though this isn't meant to go in this repo a message here would be great such as:
`Unable to automatically initalize TestBed.
  You must manually initalize TestBed if you intend to use it in your tests.
  Error:
  ${stack tace / message}
`

Reply:
Yeah we should probably put that in the example repo.